### ISSUE TITLE
[Build] Failing silently in 2.4.1

### ISSUE THREAD

**Issue by eoliphan** [Semantic-Org/Semantic-UI#6622]
### Steps
run `gulp build`

### Expected Result
Output CSS bundle

### Actual Result
No CSS, the build process stops after creation of checkbox.min.css.  Reverting to 2.4.0 works with no issues
```
[09:35:55] Created: ../resources/public/css/components/accordion.min.css
[09:35:55] Created: ../resources/public/css/components/checkbox.css
[09:35:55] Created: ../resources/public/css/components/checkbox.min.css
Erichs-MBP:semantic ErichOliphant$ 
```

### Version
2.4.1

**Comment by ColinFrick** [comment]
I can confirm this problem

**Comment by himanshu230** [comment]
Hi All,
I have made a PR which solves the issue #6631

**Comment by jlukic** [comment]
I've been on vacation in Ireland from about an hour after releasing `2.4.1` and now. 

Will take a look at issue and related PR

**Comment by jlukic** [comment]
I'm unable to reproduce the issue. I'm currently able to build all css with `node -v` `8.9.0`

The steps I'm using to reproduce
1) Create new directory `fake/`
2) Run `npm install --save semantic ui`
3) `cd semantic` then `gulp build`
4) Confirm all files are in `dist/` and confirm version is `2.4.1`

**Comment by jlukic** [comment]
Comments left for fix on #6631

**Comment by jlukic** [comment]
`2.4.2` released with fix
